pages/pricing/_pricing-faqs.html

```diff
@@ -1,6 +1,6 @@
 <div id="pricing-faqs" class="row row-page-header">
   <div class="col-xs-12">
-    <h2 class="text-center">Pricing FAQ's</h2>
+    <h2>Pricing FAQ's</h2>
```

Comment:
I think this actually looks better centered for now.

Reply:
Yes